src/sugar3/graphics/toggletoolbutton.py

```diff
@@ -76,11 +97,25 @@ def __destroy_cb(self, icon):
             self._palette_invoker.detach()
 
     def set_icon_name(self, icon_name):
+        '''
+    Shows icon for toggle tool button
+    The set_icon_name() method sets the "icon-name" property to
+    the value of icon_name. If it is none then no icon will be shown on the
+    toogle tool button.
+    Args:
+      icon_name(string): The name for a themed icon. It can be 'None' too.
+    Example:
+      def set_icon_name(self, icon_name)
```

Comment:
This is not an example. Please provide an example on invoking this method of the class, not defining it.